src/App.tsx

```diff
@@ -34,6 +34,15 @@ const useStyles = makeStyles({
     display: 'flex',
     justifyContent: 'center',
     alignItems: 'center'
+  },
+  player: {
+    fontSize: '3rem',
+    color: theme.status.secondary
+  },
+  'mt-3': {
```

Comment:
why is this class named like that?

Reply:
Secondary? Because I used the example that gave me the material and I did not change the name, I will change it to color

Reply:
No I meant mt-3, does not look descriptive enough

Reply:
Ohh you're right, sorry it was only supposed to be a class for the margin-top but I don't remember why I put the rest of it

Reply:
Changes uploaded